python/nav/web/alertprofiles/views.py

```diff
@@ -120,6 +104,45 @@ def overview(request):
     return render(request, 'alertprofiles/account_detail.html', info_dict)
 
 
+def groups_and_permissions_modal(request):
+    """Render a modal with information about groups and permissions"""
+    account = get_account(request)
+
+    # Get information about user
+    groups = account.groups.all()
+    active_profile = _get_active_profile(account)
+
+    # Get information about users privileges
+    sms_privilege = account.has_perm('alert_by', 'sms')
+    filter_dict = {'group_permissions__in': [g.id for g in groups]}
+    filter_groups = FilterGroup.objects.filter(**filter_dict).order_by('name')
+
+    language = account.preferences.get(account.PREFERENCE_KEY_LANGUAGE, 'en')
+    language_form = LanguageForm(initial={'language': language})
+
+    return render_modal(
+        request,
+        'alertprofiles/_groups_and_permissions_modal.html',
+        context={
+            'active_profile': active_profile,
+            'filter_groups': filter_groups,
+            'groups': groups,
+            'language_form': language_form,
+            'sms_privilege': sms_privilege,
+        },
+        modal_id="groups-and-permissions",
+        size="large",
+    )
+
+
+def _get_active_profile(account: Account):
```

Comment:
I don't think we need this extra function since `Account.get_active_profile()` already catches `AlertPreference.DoesNotExist` and `AlertProfile.DoesNotExist` exceptions

Reply:
Ah alright. I assumed it did not work this way, since the existing code had an explicit `DoesNotExist` check. Removed it 😄